cli/cmd/vuln_container_list_assessments.go

```diff
@@ -309,15 +309,17 @@ func listVulnCtrAssessments(
 		treeOfContainerVuln.ParseData(response.Data)
 	}
 
-	cli.Log.Infow("evaluation guids", "count", len(treeOfContainerVuln.ListEvalGuid()))
-	if len(treeOfContainerVuln.ListEvalGuid()) != 0 {
+	evalGuids := treeOfContainerVuln.ListEvalGuid()
```

Comment:
This is not part of the fix, but ListEvalGuid() was previously being called 3 times below.